_chapters/1.home.md

```diff
@@ -71,6 +71,24 @@ Test the asset against conventions held within an organisation to prevent bad co
 <br>
 <br>
 
+### What plugins are available?
+
+- Official plugins
+  - https://github.com/pyblish/pyblish-starter
+  - https://github.com/pyblish/pyblish-advance
+  - https://github.com/pyblish/pyblish-flower
+  - https://github.com/pyblish/pyblish-magenta
```

Comment:
How about we give these names rather than links, e.g. [Magenta](https://github.com/pyblish/pyblish-magenta) as opposed to https://github.com/pyblish/pyblish-magenta

Reply:
Done :)